testsuite/tests/kuadrant/authorino/identity/plain/test_plain_identity.py

```diff
@@ -0,0 +1,23 @@
+"""Test for plain identity"""
+
+import pytest
+
+pytestmark = [pytest.mark.authorino]
+
+
+def test_plain_identity(client, authorization):
+    """
+    Test:
+        - Add Plain Identity using nonexistent path in auth_json as selector
+        - Send request
+        - Assert that response status code is 401 (Unauthorized)
+        - Replace Plain Identity selector value with existent path in auth_json
+        - Assert that response status code is 200 (OK)
+    """
+    authorization.identity.add_plain("nonexistent-path", "auth.identity")
+    response = client.get("/get")
+    assert response.status_code == 401
+
+    authorization.identity.add_plain("existent-path", "context.request.http.method")
```

Comment:
This will not replace the existing plain identity but will put it alongside the previous "nonexistent-path". This will cause the `auth.identity` to contain the method verb `GET` and both the previous and this will pass with result authenticated.

There is no clean way to replace previously created identities other than using `authorization.identity.clear_all()` which will remove all.

Reply:
Hey, I should have checked that this only adds stuff and not replace stuff. I refactor the test to use parametrized fixture instead of changing the authconfig in test directly which solves the issue. Thanks for pointing this out.